src/routes/controllers/users/getUserInfo.js

```diff
@@ -1,5 +1,7 @@
 const getUserFromToken = require('../../../helpers/getUserFromToken');
+const checkReviewability = require('../../../helpers/checkReviewability');
 const { pool } = require('../../../utils/db');
+const { check } = require('prettier');
```

Comment:
I'm guessing that got added in automatically when I was calling the helper function and accidentally wrote check() once. I'll just get rid of it though because it might confuse us later.